lib/db.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "icon": "🚜",
+    "name": "Tractor"
+  },
+  {
+    "icon": "🐴",
+    "name": "Horse"
+  },
+  {
+    "icon": "🌻",
+    "name": "Sunflower"
+  },
+  {
+    "icon": "🌈",
+    "name": "Rainbow"
+  },
+  {
+    "icon": "🦌",
+    "name": "Deer"
+  },
+  {
+    "icon": "🐗",
+    "name": "Wild boar"
+  },
+  {
+    "icon": "🎡",
+    "name": "Ferris wheel"
+  },
+  {
+    "icon": "🛝",
+    "name": "Slide"
+  },
+  {
+    "icon": "🍓",
+    "name": "Strawberry"
+  }
+]
```

Comment:
why do you have a db.json and db.js?

Reply:
I Want to use MongoDB Atlas later on, so I have a default .json to setup the collection.